package.json

```diff
@@ -4,19 +4,22 @@
   "author": "Lance Stout <lance@andyet.net>",
   "bugs": "https://github.com/otalk/jingle-media-session/issues",
   "contributors": [
-    "Philipp Hancke <fippo@andyet.net>"
+    "Philipp Hancke <fippo@goodadvicepages.net>",
```

Comment:
just remove the email please. This one is wrong but the real one gets enough spam as is